### PR TITLE
🧑‍💻 add team member

### PR DESCRIPTION
**Pull request by smolgroot**
Just added Megan into the team members list.

### CODE REVIEW

**Review by mrtingalingling — APPROVED** [review]
LGTM!

**Comment by mrtingalingling** [comment]
Let's wait for Megan's confirmation before we merge

**Comment by clandestinecurio** [comment]
Yep, looks good. Thank you!

**Pull request merged**
